whois-api/src/main/java/net/ripe/db/whois/api/httpserver/RemoteAddressCustomizer.java

```diff
@@ -0,0 +1,43 @@
+package net.ripe.db.whois.api.httpserver;
+
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import com.google.common.collect.Iterables;
+import com.google.common.net.HttpHeaders;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.Request;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import java.net.InetSocketAddress;
+import java.util.Enumeration;
+
+public class RemoteAddressCustomizer  implements HttpConfiguration.Customizer {
```

Comment:
Maybe add a comment to explain that X-Forwarded-For address will replace the request address.

Reply:
Thanks!